source/scripting/Game.cs

```diff
@@ -156,19 +156,29 @@ public static float TimeScale
 			}
 		}
 		/// <summary>
-		/// returns the instantaneous Frames per Second
+		/// Gets the total number of frames that's been rendered in this session.
 		/// </summary>
-		public static float FPS
+		public static int FrameCount
 		{
 			get
 			{
-				return 1.0f / LastFrameTime;
+				return Function.Call<int>(Hash.GET_FRAME_COUNT);
 			}
 		}
 		/// <summary>
-		/// Gets how long(in seconds) the last frame was displayed before the next was rendered
+		/// Gets the current frame rate per second.
 		/// </summary>
-		public static float LastFrameTime
+		public static float FrameRate
+		{
+			get
+			{
+				return 1.0f / DeltaTime;
+			}
+		}
+		/// <summary>
+		/// Gets the time it currently takes to render a frame, in seconds.
+		/// </summary>
+		public static float DeltaTime
```

Comment:
What's wrong with `LastFrameTime`? Yes, `DeltaTime` means the same, but the former is more obvious.